Makefile

```
@@ -465,6 +465,14 @@ check-appearance:
 	tests/test-dwm-settings-theme.sh
 	tests/test-dwm-xsettings.sh
 
+check-phase5-optional-components:
```

checkmake (0.3.2)

[warning] Target body for "check-phase5-optional-components" exceeds allowed length of 5 lines (6).

(maxbodylength)